lib/bindings/src/breez_sdk_liquid.udl

```diff
@@ -313,35 +325,47 @@ dictionary GetInfoResponse {
 };
 
 dictionary PrepareSendRequest {
-    string invoice;
+    string payment_destination;
+    u64? amount_sat;
+};
+
+[Enum]
+interface SendDestination {
+    BIP21(LiquidAddressData address_data);
```

Comment:
perhaps rename to `LiquidAddress(LiquidAddressData address_data);`
So we are consistent with the receive ?